criteo_distributed_training/autoscalar_aws.yml

```diff
@@ -165,18 +170,18 @@ setup_commands:
 
 # Custom commands that will be run on the head node after common setup.
 head_setup_commands:
-  - pip3 install scikit-learn
-  - git clone https://github.com/ThirdAILabs/Public-Benchmarks.git
+    - pip3 install scikit-learn
+#   - git clone https://github.com/ThirdAILabs/Public-Benchmarks.git
 
 # Custom commands that will be run on worker nodes after common setup.
 worker_setup_commands: []
 
 # Command to start ray on the head node. You don't need to change this.
 head_start_ray_commands:
     - ray stop
-    - ray start --head --port=6379 --object-manager-port=8076 --autoscaling-config=~/ray_bootstrap_config.yaml
+    - ray start --head --port=6379 --object-manager-port=8076 --autoscaling-config=~/ray_bootstrap_config.yaml --system-config='{"core_worker_internal_heartbeat_ms":10000}'
```

Comment:
Could you add a comment, why we are adding it?

Reply:
Added. 
Please check whether it conveys the idea well....